src/basic/meson.build

```diff
@@ -329,6 +329,7 @@ libbasic = static_library(
         dependencies : [versiondep,
                         threads,
                         libcap,
+                        libseccomp,
```

Comment:
I'd very much prefer to move that code to src/shared/ and not introduce the dependency here.

Reply:
yupp, this is a bug. the relevant functions should be moved to src/shared/

@topimiettinen